homeassistant/components/geocaching/const.py

```diff
@@ -26,3 +28,9 @@
 }
 
 ENVIRONMENT = GeocachingApiEnvironment.Production
+
+PLATFORMS = [Platform.SENSOR]
+
+# Sensor ID formats
+PROFILE_ID_SENSOR_FORMAT = DOMAIN + ".{}_{}"
+CACHE_ID_SENSOR_FORMAT = DOMAIN + ".{}_{}"
```

Comment:
Seems unneeded, additionally they are incorrect (domain should not be part of unique IDs).